src/include/processor/operator/copy/copy_node.h

```diff
@@ -115,8 +111,16 @@ class CopyNode : public Sink {
     }
 
 protected:
-    std::unique_ptr<CopyNodeLocalState> localState;
     std::shared_ptr<CopyNodeSharedState> sharedState;
+    CopyNodeDataInfo copyNodeDataInfo;
+    common::CopyDescription copyDesc;
```

Comment:
Our current design is to wrap all these static information into a wrapper class called xxxInfo